documentation/slate/source/includes/welcome.md

```diff
@@ -15,11 +15,20 @@ It provides the following services
 
 navitia is a [HATEOAS](http://en.wikipedia.org/wiki/HATEOAS) API that returns JSON formated results, available at <https://api.navitia.io>
 
-Have a look at the examples below to learn what services we provide and how to use them.
+Have a look at the examples below to learn what services we provide and how to use them. 
+
+<aside class="notice">
+A fake token is used in examples below: be aware that this token is really limited. You should use yours to get better services on real data.
+</aside>
 
 Getting started
 ---------------
 
+``` shell
+$ curl 'https://api.navitia.io/v1/coverage/sandbox/' -H 'Authorization: 3b036afe-0110-4202-b9ed-99718476c2e0'
```

Comment:
It is a bit abrupt as the first example. 
maybe we can add some explanation above ?